README.md

```diff
@@ -145,14 +145,14 @@ You might want to get some information about the location that corresponds to a
 ```pycon
 >>> from phonenumbers import geocoder
 >>> ch_number = phonenumbers.parse("0431234567", "CH")
->>> print repr(geocoder.description_for_number(ch_number, "de"))
-u'Z\\xfcrich'
->>> print repr(geocoder.description_for_number(ch_number, "en"))
-u'Zurich'
->>> print repr(geocoder.description_for_number(ch_number, "fr"))
-u'Zurich'
->>> print repr(geocoder.description_for_number(ch_number, "it"))
-u'Zurigo'
+>>> print(repr(geocoder.description_for_number(ch_number, "de")))
+'Z\\xfcrich'
```

Comment:
I think this comes out as `'Zürich'` in Python 3.

Reply:
Right. I'll also remove the now unecessary `print(repr(` clutter.